src/scenario.cpp

```diff
@@ -647,6 +644,15 @@ char *clean_cdata(char *ptr, int *removed_crlf = NULL)
         memmove(ptr, ptr + 1, strlen(ptr));
     }
 
+    /* Re-add our removed (CR)LFs. */
+    while (removed_crlf--) {
+        strcat(msg, "\n");
+    }
+    /* A SIP message must contain a double (CR)LF to signify EOH. */
```

Comment:
Yeah. It's mandatory in SIP. It was an attempt to document that the CR wasn't used here. But you're right: it's better to be explicit (ooh, zen of python).